packages/camera/camera/example/ios/RunnerTests/AvailableCamerasTest.m

```diff
@@ -0,0 +1,123 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera;
+@import camera.Test;
+@import XCTest;
+@import AVFoundation;
+#import <OCMock/OCMock.h>
+#import "MockFLTThreadSafeFlutterResult.h"
+
+@interface AvailableCamerasTest : XCTestCase
+@end
+
+@implementation AvailableCamerasTest
+
+- (void)testAvailableCamerasShouldReturnAllCamerasOnMultiCameraIPhone {
+  CameraPlugin *camera = [[CameraPlugin alloc] initWithRegistry:nil messenger:nil];
+  XCTestExpectation *expectation =
+      [[XCTestExpectation alloc] initWithDescription:@"Result finished"];
+
+  // iPhone 13 Cameras:
+  AVCaptureDevice *wideAngleCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([wideAngleCamera uniqueID]).andReturn(@"0");
+  OCMStub([wideAngleCamera position]).andReturn(AVCaptureDevicePositionBack);
+
+  AVCaptureDevice *frontFacingCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([frontFacingCamera uniqueID]).andReturn(@"1");
+  OCMStub([frontFacingCamera position]).andReturn(AVCaptureDevicePositionFront);
+
+  AVCaptureDevice *ultraWideCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([ultraWideCamera uniqueID]).andReturn(@"2");
+  OCMStub([ultraWideCamera position]).andReturn(AVCaptureDevicePositionBack);
+
+  AVCaptureDevice *telephotoCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([telephotoCamera uniqueID]).andReturn(@"3");
+  OCMStub([telephotoCamera position]).andReturn(AVCaptureDevicePositionBack);
+
+  NSMutableArray *requiredTypes = [NSMutableArray array];
+  [requiredTypes addObjectsFromArray:@[
+    AVCaptureDeviceTypeBuiltInWideAngleCamera, AVCaptureDeviceTypeBuiltInTelephotoCamera
+  ]];
```

Comment:
This is both more readable and more performant (the latter doesn't matter in a test of course, but it's a better practice in general) as:

```
NSMutableArray *requiredTypes = [@[AVCaptureDeviceTypeBuiltInWideAngleCamera, AVCaptureDeviceTypeBuiltInTelephotoCamera] mutableCopy];
```

Reply:
sure 👍